splice_access_api/src/main/java/com/splicemachine/access/configuration/AuthenticationConfiguration.java

```diff
@@ -54,7 +54,7 @@
     // In this particular instance, there are Splice beta customers with AnA disabled and they want to
     // switch to using native AnA.  So we allow a manual override here.  See DB-2088 for more details.
     public static final String AUTHENTICATION_NATIVE_CREATE_CREDENTIALS_DATABASE = "splice.authentication.native.create.credentials.database";
-    private static final boolean DEFAULT_AUTHENTICATION_NATIVE_CREATE_CREDENTIALS_DATABASE = true;
+    private static final boolean DEFAULT_AUTHENTICATION_NATIVE_CREATE_CREDENTIALS_DATABASE = false;
```

Comment:
There is also an xml property associated: splice.authentication.native.create.credentials.database, do we need to change that too as a default?